server/seenomaly/lambda/constants.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+ROOT_PATH = "." #Relative path should work fine, if issues arrise set to the absoulute path of the Seenomaly containing directory.
+UNQUALIFIED_DATA_PATH = "data"
```

Comment:
What does UNQUALIFIED_DATA_PATH mean?

Reply:
I don't really know tbh haha, looks like something they had when developing the algorithm

Reply:
It's just the name of the folder (or subfolder) of where the data is stored. It's unqualified as it isn't valid until being combined with the root path on the next line. This is only used when testing locally.